### DISCUSSION TITLE
Can I connect to an already running measurement?

### DISCUSSION THREAD

**Discussion by martinjthompson** [chaitu-ycr/py_canoe#60] [Q&A] [answered]
I would like to avoid the time consuming setup part of the process, and just connect to a running measurement ... is this possible (at least in theory?), or is there some technical blocker to achieving this?

**Comment by anandautam1** [comment, marked as answer]
I think this can be done via - [This API](https://github.com/chaitu-ycr/py_canoe/blob/7a990f09fb985a85717f7dda44bdacdae8fc4c70/src/py_canoe/core/application.py#L157)
